operator-pipeline-images/operatorcert/integration/testcases/basic.py

```diff
@@ -0,0 +1,82 @@
+"""
+Basic test cases
+"""
+
+import logging
+from pathlib import Path
+from shutil import rmtree
+from tempfile import mkdtemp
+from typing import Optional
+
+from git import Repo
+
+from operatorcert.integration.config import Config
+from operatorcert.integration.testcase import BaseTestCase, integration_test_case
+
+
+@integration_test_case
+class AddBundle(BaseTestCase):  # pragma: no cover
+    """
+    Test the addition of a non-FBC bundle
+    """
+
+    def __init__(self, config: Config, logger: logging.Logger) -> None:
+        super().__init__(config, logger)
+        self.tempdir: Optional[Path] = None
+
+    def setup(self) -> None:
+        self.tempdir = Path(mkdtemp(prefix=self.run_id + "-"))
+        test_branch = "add-bundle"
+        remote_branch = f"test-{self.run_id}"
+        git_fixture_dir = self.tempdir / "git-fixture"
+        # clone fixture repo locally
+        local_repo = Repo.clone_from(
+            self.config.fixtures_repository.url, git_fixture_dir, branch=test_branch
+        )
+        # configure remotes
+        local_repo.create_remote("operators", url=self.config.operator_repository.url)
+        local_repo.create_remote(
+            "contributor", url=self.config.contributor_repository.url
+        )
+        # The fixture branch contains the operator structure needed for the test
+        # and the latest commit is the change being tested. This means we need
+        # to exclude the latest commit when pushing to the operator repo.
+        local_repo.create_head(
```

Comment:
This is not something that should be fixed as part of this PR, but we should be able to dynamically change a version of an operator instead of using a hardcoded version for all test-cases.

Reply:
This creates the branch in the local (ephemeral) copy of the repo just for convenience in the push step below; it's isolated from other tests so there is no chance of collision.